<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyDialog</class>
 <widget class="QDialog" name="PropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件属性</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labico">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labName">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>351</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件状态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>下载地址</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labsize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labtype">
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labstate">
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineLocation">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>271</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>271</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOpen">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string>打开文件夹(&amp;O)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineurl">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>271</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>271</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCopy">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string>复制(&amp;C)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton[highlight=&quot;true&quot;]{
	color: white;
	background-color: rgb(0,103,192);
	border:1px solid rgb(20,115,197);
	border-bottom: 2px solid rgb(0,62,115);
}
QPushButton[highlight=&quot;true&quot;]:disabled{
	background-color: rgb(3, 41, 74);
}
QPushButton[highlight=&quot;true&quot;]:hover{
	color: white;
	background-color: rgb(25,117,197);
	border:1px solid rgb(43,128,202);
	border-bottom: 2px solid rgb(15,70,118);
}
QPushButton[highlight=&quot;true&quot;]:pressed{
	color: rgb(194,218,239);
	background-color: rgb(25,117,197);
	border: 0px ;
}</string>
       </property>
       <property name="text">
        <string>确定(&amp;O)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>PropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
